src/main/java/pl/kruczala/michal/Casket50.java

```diff
@@ -0,0 +1,26 @@
+package pl.kruczala.michal;
+
+public class Casket50 {
```

Comment:
Ogólnie jest ok, możesz mieć klasy dla każdego banknotu osobno, bo banknoty raczej się nie zmieniają często natomiast w Wenezueli miałbyś już problem bo tam wchodzą naprawdę dziwne nominały więc musiałbyś produkować wiele klas typu Casket100000 itd. Natomiast, możesz połączyć te wszystkie klasy w jedną klasę Casket która w konstruktorze przyjmie nominał na jakim operuje. Wtedy możesz mieć i 100 różnych banknotów. Z punktu widzenia kasetki nic się nie zmienia. Innym powodem przemawiającym za połączeniem tych klas w jedną jest to, że kod wygląda praktycznie tak samo a w programowaniu jest taka zasada DRY (Don't Repeat Yourself) więc mógłbyś mieć jeden algorytm który wydaje banknoty tak samo, jedynie co się dla niego zmienia to nominał. wtedy z Casket możesz zamiast int zwracać np. coś takiego
```
public class CasketResult
{
    public int nominal;
    public int banknotesCount;
}
```